eZ/Publish/Core/settings/fieldtypes.yml

```diff
@@ -431,6 +423,14 @@ services:
 
     # Not implemented fieldtypes
     # Configured to use the Null type to not throw exception
+
+    ezpublish.fieldType.ezpage:
```

Comment:
If the reason for this is some test data, then we need to get rid of them and maybe provide upgrade script. Otherwise we didn't really drop this FT.